packages/element/src/serialize.js

```diff
@@ -219,6 +219,46 @@ const CSS_PROPERTIES_SUPPORTS_UNITLESS = new Set( [
 	'zoom',
 ] );
 
+/**
+ * Returns a string with ampersands escaped. Note that this is an imperfect
```

Comment:
When I read 'imperfect' it makes me think that the function is unfinished, when in fact it's _intentionally_ imperfect.

Reply:
>When I read 'imperfect' it makes me think that the function is unfinished, when in fact it's intentionally imperfect.

Depends how you view it. Technically it could be improved, but as I saw it the only option would be to include a [full list of valid named character references](https://w3c.github.io/html/syntax.html#named-character-references), and in weighing the cost in doing so, I thought it to not have a net positive benefit.